GCDWebServer/Core/GCDWebServer.h

```diff
@@ -217,6 +217,10 @@ extern NSString* const GCDWebServerAuthenticationMethod_DigestAccess;
 - (void)webServerDidCompleteBonjourRegistration:(GCDWebServer*)server;
 
 /**
+ *  This method is called if Bonjour registration were failed for any reason */
```

Comment:
Should be "This method is called if the Bonjour registration fails for any reason." and the `*/` should be on the next line like the other documentation entries.